src/ctia/task/migration/store.clj

```diff
@@ -188,15 +203,19 @@
 
 (s/defn store-map->es-conn-state :- ESConnState
   "Transforms a store map in ES lib conn state"
-  [conn-state :- StoreMap]
-  (dissoc (clojure.set/rename-keys conn-state {:indexname :index})
-          :mapping :type :settings))
-
-(defn bulk-metas
+  [conn-state :- StoreMap
+   services :- MigrationStoreServices]
+  (-> conn-state 
+      (set/rename-keys {:indexname :index})
+      (dissoc :mapping :type :settings)
+      (assoc :services (MigrationStoreServices->ESConnServices
+                         services))))
```

Comment:
The above change adds the `:services` key, otherwise is a refactoring to use `->`.